package.json

```diff
@@ -22,7 +22,7 @@
   "homepage": "https://github.com/electricimp/Builder#readme",
```

Comment:
Please bump up the Builder version up to 2.4.0

Reply:
Updated